docs/sphinx/install.rst

```diff
@@ -3,9 +3,16 @@
 Installation
 ============
 
-The histogram package can be installed by using easy_install::
+The histogram package can be installed by using pip install::
 
- $ easy_install histogram
+ Download the package at https://github.com/neutrons/histogram-dev.
```

Comment:
Download the source code from

Reply:
updated this and fixedthe link of the logo image